.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file required, or is it really a separate change?

Reply:
I don't think so, it looks like this was created by intellij and I didn't filter it out.